src/stores/PostStore.ts

```diff
@@ -148,6 +146,17 @@ farcasterContract.on(
     PostIdsStatuses.statuses[id.toNumber()] = Promise.resolve(
       PostStatus.pending
     )
+
+    /* Check if sender is the `replayAll` address */
+    const replyToAll = await QuestionOfDayStore.qodAddress
+    if (sender !== replyToAll) return
+
+    /* Add new post as QoD */
+    const allQodPostIds = await QuestionOfDayStore.allQodPostIds
+    QuestionOfDayStore.allQodPostIds = Promise.resolve([
+      ...allQodPostIds,
+      id.toNumber(),
```

Comment:
Let's place this above the `...alQodPostIds` and remove the comment, so everyone will see, that you're copying previous results and adding new id